packages/atomic/src/components/ipx/atomic-ipx-button/atomic-ipx-button.tsx

```diff
@@ -38,14 +43,23 @@ export class AtomicIPXButton {
         onClick={() => this.onClick()}
       >
         <span part="button-icon">
-          <atomic-icon
-            part="ipx-close-icon"
-            icon={this.getIcon(CloseIcon)}
-          ></atomic-icon>
-          <atomic-icon
-            part="ipx-search-icon"
-            icon={this.getIcon(SearchIcon)}
-          ></atomic-icon>
+          {this.icon ? (
```

Comment:
Instead of having multiple conditions here and rendering multiple buttons, I think it would be easier to read if `CloseIcon` and `SearchIcon` were passed down from the parent component to this component's `icon` property, and if the `part` was a prop.

Reply:
I like the idea to have these two as prop of the parent. I'll try it